examples/webgpu_tsl_editor.html

```diff
@@ -0,0 +1,218 @@
+<html lang="en">
+	<head>
+		<title>three.js - webgpu - tsl editor</title>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0">
+		<link type="text/css" rel="stylesheet" href="main.css">
+	</head>
+	<body>
+
+		<style>
+			#source {
+				position: absolute;
+				top: 0;
+				left: 0;
+				width: 50%;
+				height: 100%;
+			}
+			#result {
+				position: absolute;
+				top: 0;
+				right: 0;
+				width: 50%;
+				height: 100%;
+			}
+			#renderer {
+				position: absolute;
+				top: 0;
+				right: 50%;
+				width: 200px;
+				height: 200px;
+				z-index: 100;
+				pointer-events: none;
+			}
+		</style>
+
+		<div id="source"></div>
+		<div id="result"></div>
+		<div id="renderer"></div>
+
+		<script async src="https://unpkg.com/es-module-shims@1.6.3/dist/es-module-shims.js"></script>
+
+		<script src="https://cdn.jsdelivr.net/npm/monaco-editor@latest/min/vs/loader.min.js"></script>
+
+		<script type="importmap">
+			{
+				"imports": {
+					"three": "../build/three.module.js",
+					"three/addons/": "./jsm/",
+					"three/nodes": "./jsm/nodes/Nodes.js"
+				}
+			}
+		</script>
+
+		<script type="module">
+
+			import * as THREE from 'three';
+			import * as NODES from 'three/nodes';
```

Comment:
I think we used to import nodes as `Nodes`?

Reply:
I noticed certain users using `NODES`, I have doubts what to do here, all namespaces are uppercase by default, should we keep it or change it? @mrdoob